aws.cabal

```diff
@@ -139,7 +139,7 @@ Library
                        scientific           >= 0.3,
                        tagged               >= 0.7     && < 0.9,
                        text                 >= 0.11,
-                       time                 >= 1.1.4   && < 2.0,
+                       time                 >= 1.4.0   && < 1.7,
```

Comment:
Which ghc version has time 1.4? I wonder if we might go further and make this time 1.5+ only. Also, why the tightening of the upper bound?